### PR TITLE
Update to latest Dart and Flutter

### PR DESCRIPTION
**Pull request by AndreTheHunter**
Update to latest flutter and dart https://docs.flutter.dev/release/archive
- [ ] run `build.sh`

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -3,6 +3,8 @@ set -euo pipefail
 
 docker images -a | grep "anvil" | awk '{print $3}' | xargs docker rmi --force || true
 
+export DOCKER_DEFAULT_PLATFORM=linux/amd64
```
Comment (AndreTheHunter) [inline build.sh:6]: Makes it work on M1 macs

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ A Docker Development Build and Test Container where Projects are hammered into s
 
 ![Docker Pulls](https://img.shields.io/docker/pulls/axrs/anvil?style=for-the-badge)
 
-Base Image [Debian:bullseye-slim](https://hub.docker.com/_/debian)
+Base Image [Debian:stable-slim](https://hub.docker.com/_/debian). See [Debian Releases](https://www.debian.org/releases/)
```
Comment (AndreTheHunter) [inline README.md]: Sable is bullseye ATM

**Review by AndreTheHunter — COMMENTED** [review]
I tried to `build.sh` but ran out of disk space

File: README.md (hunk 2 above)
Comment (axrs) [reply README.md]: Stable is a tag that changes over time. These containers should be stable in that a major OS bump shouldn't just occur. Especially if users have setup package repos that target bullseye specifically.

File: Anvil.Dockerfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
-FROM debian:bullseye-slim
-LABEL maintainer="Alexander Scott <xander@axrs.io>"
+FROM debian:stable-slim
+LABEL maintainer="Alexander Scott <xander@axrs.io>, André de Jager <andrethehunter@gmail.com>"
```
Comment (axrs) [inline Anvil.Dockerfile]: Move to a secondary label: 'contributors'

**Review by axrs — CHANGES_REQUESTED** [review]

**Comment by axrs** [comment]
I'll make some changes and get some new containers deployed

**Pull request merged**